includes/db_connection.php

```diff
@@ -1,10 +1,14 @@
 <?php
-	// mysqli connection
-	//require_once('includes/settings.php');
```

Comment:
@MehaKaushik You have commented on the wrong section of the file. That's the old code, which I have already removed.
<img width="1275" alt="screen shot 2017-06-07 at 23 03 55" src="https://user-images.githubusercontent.com/13470643/26892387-b6954fdc-4bd5-11e7-9e12-14cb618734d9.png">